cvat-ui/src/actions/tasks-actions.ts

```diff
@@ -519,3 +524,86 @@ export function hideEmptyTasks(hideEmpty: boolean): AnyAction {
 
     return action;
 }
+
+export function showMoveTaskModal(taskInstance: any): AnyAction {
+    const action = {
+        type: TasksActionTypes.SHOW_MOVE_TASK_MODAL,
+        payload: {
+            taskInstance,
+        },
+    };
+
+    return action;
+}
+
+export function closeMoveTaskModal(): AnyAction {
+    const action = {
+        type: TasksActionTypes.CLOSE_MOVE_TASK_MODAL,
+    };
+
+    return action;
+}
+
+// TODO: should be options type here?
```

Comment:
Is it relevant TODO?

Reply:
No, it was relevant to the previous feature implementation. Deleted.